src/Command/PullRequest/PullRequestPatOnTheBackCommand.php

```diff
@@ -48,11 +81,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $adapter = $this->getAdapter();
         $pr = $adapter->getPullRequest($prNumber);
+        if (!$pat = $input->getOption('pat')) {
+            $pat = null;
+        }
+
+        $config = $this->getConfig();
+        if (!$pats = $config->get('pats')) {
+            $pats = Pats::getDefaultPats();
+        }
+
+        if ($input->getOption('random')) {
```

Comment:
We should probably not allow `--random` and `--pat` to be passed at the same time, since it won't make sense to use both, so there should be an exception if both is passed